platform/sun50i/include/platform/r_ccu.h

```diff
@@ -20,6 +20,7 @@
 
 #define R_CCU_RESET_BASE    (0x00b0 / sizeof(uint32_t))
 
+#define R_CCU_RESET_R_PIO   BITMAP_INDEX(R_CCU_GATE_BASE + 0, 0)
```

Comment:
I already defined all of the gates/resets when adding the `R_CCU`. If there isn't one listed, there isn't one in the hardware.